samples/config/config.json

```diff
@@ -0,0 +1,77 @@
+{
+  "enclave_file": "lib_jsgeneric.enclave.so.signed",
+  "enclave_type": "release",
+  "network": {
+    "node_address": { "hostname": "127.0.0.1", "port": "0" },
+    "rpc_interfaces": [
+      {
+        "rpc_address": { "hostname": "127.0.0.1", "port": "0" },
+        "public_rpc_address": { "hostname": "ccf.dummy.com", "port": "12345" },
+        "max_open_sessions_soft": 1000,
+        "max_open_sessions_hard": 1010
+      }
+    ]
+  },
+  "node_certificate": {
+    "subject_name": "CN=CCF Node",
+    "subject_alt_names": ["IPAddress:127.0.0.1", "dNSName:ccf.dummy.com"],
+    "curve_id": "secp384r1",
+    "initial_validity_period_days": 1
+  },
+  "start": {
+    "constitution_files": [],
+    "members": [
+      {
+        "certificate_file": "member0_cert.pem",
+        "encryption_public_key_file": "member0_enc_pubk.pem",
+        "data_json_file": ""
+      }
+    ],
+    "service_configuration": {
+      "recovery_threshold": 0,
+      "max_allowed_node_cert_validity_days": 365
+    }
+  },
+  "join": {
+    "join_timer_ms": 1000,
+    "target_rpc_address": { "hostname": "127.0.0.1", "port": "0" }
+  },
+  "ledger": {
+    "ledger_dir": "ledger",
```

Comment:
Here and elsewhere: should this simply be `"dir"` since it's already in the `"ledger"` object?